### DISCUSSION TITLE
Color in LaTeX Output

### DISCUSSION THREAD

**Discussion by Fiveby5** [rstudio/gt#1300] [Q&A]
Hi,

I am making PDF output reports in Quarto, and I've been using flextable since I can color code things in the table using that package. I use gt when I'm making html reports though, and much prefer it. Is there a way to get the colors through data_color(), opt_stylize(), style() etc to render to pdf as well?  I've tried adding latex packages to the header (amsmath, booktabs, caption, longtable, colortbl, color), but still no success. 

Thanks!

**Comment by NorthernDruid** [comment]
this would indeed be amazing!

**Comment by olivroy** [comment]
This should already be possible since gt 0.11.0 and 0.11.1. Please try and report back! `data_color()` and `tab_style()` work out of the box!

`opt_stylize()` doesn't wokr.

In a Quarto doc,

````qmd

---
format: pdf
---

```{r}
library(gt)
exibble |> gt() |> data_color()
```
````

![image](https://github.com/user-attachments/assets/dc4d5a8c-95b5-4407-9e40-3d542afc708a)
